src/components/interfaces/MOBILE_API.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" standalone="no"?>

```

Comment:
@dev-gh formatting should be done in this file. You could view actual changes using ignore whitespace flag. See this commit using this link ab9adc7?w=1